src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -40,10 +40,10 @@ export const SchedulePageModal: FC<SchedulePageModalProps> = ({
     withEvents: true,
   });
 
-  const relatedVenuesById: Record<
+  const relatedVenuesById: Partial<Record<
     string,
     WithId<AnyVenue>
-  > = relatedVenues.reduce(itemsToObjectByIdReducer, {});
+  >> = relatedVenues.reduce(itemsToObjectByIdReducer, {});
```

Comment:
Highlighting this as it's the main type related change in this PR.

It's quite a localised fix, but I believe this will be a problem with our root Firestore types as per https://github.com/sparkletown/sparkle/pull/744, so we should solve it there, and make sure to fix all the places it surfaces as potential issues.

Reply:
Let’s get #744 finished and reviewed ASAP.

Reply:
BIG yes!